scripts/build_utils.sh

```diff
@@ -2,7 +2,7 @@
 
 set -ex
 
-VENV="$WORKSPACE/venv/bin"
+VENV="/tmp/jenkins-venv/bin"
```

Comment:
This is not only important for avoiding hard-coding, but it is entirely possible that we get into trouble on a subsequent build attempting to use the same venv